spec/elmtest_spec.vim

```diff
@@ -31,4 +31,19 @@ describe "elm-test"
     Expect g:test#last_command == 'elm-test'
   end
 
+  it "runs tests against absolute path of npm executable (elm-test)"
+    let g:test#javascript#elm#executable = 'node_modules/.bin/elm-test'
```

Comment:
Instead of setting this variable could you test this by creating `node_modules/.bin/elm-test` using `touch` like in cucumberjs_spec.vim